docs/BreakingChanges/025 EF version must match NetFX version.md

```diff
@@ -0,0 +1,22 @@
+## 25: EF version must match NetFX version
```

Comment:
{
  "Id": "25",
  "Title": "EF version must match NetFX version",
  "Details": "The entity framework version should be matched with the .NET framework version. Entity Framework 5 is recommended for .NET 4.5. There are some known issues with EF 4.x in a .NET 4.5 project around `System.ComponentModel.DataAnnotations`. In .NET 4.5, these were moved to a different assembly, so there are issues determining which annotations to use.",
  "Suggestion": "Upgrade to EF 5 for .NET 4.5",
  "Link": "http://forums.asp.net/t/1779042.aspx",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": true,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": true,
  "ImpactScope": 1
}

**Parsed APIs:**
[ ] Not detectable via API analysis